lib/mllp/sender.ex

```diff
@@ -343,6 +314,48 @@ defmodule MLLP.Sender do
     stop_connection(state, reason, "process terminated")
   end
 
+  @should_reconnect_errors [
+    :closed,
+    :enotconn,
+    :econnreset,
+    :enetreset,
+    :econnaborted,
+    :econnrefused
+  ]
```

Comment:
On this list of error, I glanced through some errors and found these which looks like we should include for reconnect. what do you think?
```
enetdown - Network is down
enetreset - Network dropped connection on reset (already there)
enetunreach - Network is unreachable
ehostdown - Host is down
ehostunreach - Host is unreachable
ecomm - Communication error on send
```
https://www.erlang.org/docs/23/man/inet.html#error_codes

Reply:
Those make sense to me, will add.

Reply:
We decided on not keep error list for retry and will instead will try to reconnect. We will close old connection though before attempting the connection again.